mavros/include/mavros/mavros_uas.h

```diff
@@ -330,11 +330,21 @@ class UAS {
 	static std::string str_system_status(enum MAV_STATE st);
 
 	/**
-	 * @brief Function to match the received orientation received by DISTANCE_SENSOR msg
+	 * @brief Function to match the received orientation received by MAVLink msg
 	 *        and the rotation of the sensor relative to the FCU.
 	 */
 	static Eigen::Quaterniond sensor_orientation_matching(MAV_SENSOR_ORIENTATION orientation);
 
+	/**
+	 * @brief Retrieve alias of the orientation received by MAVLink msg.
+	 */
+	static std::string str_sensor_orientation(MAV_SENSOR_ORIENTATION orientation);
+
+	/**
+	 * @brief Retrieve sensor orientation number from alias name.
+	 */
+	static int orientation_from_str(std::string sensor_orientation);
```

Comment:
> Change `std::string sensor_orientation` to `const std::string &sensor_orientation`.

Why? And why not also `Eigen::Quaterniond sensor_orientation_matching`?

> Also this function should be capable convert numeric value of enum (e.g. strtol() then check & return).

Convert to what? `MAV_SENSOR_ORIENTATION orientation` is supposed to be a number, which is used as an index to return: orientation name in case of `UAS::str_sensor_orientation()`; and orientation values in case of `UAS::sensor_orientation_matching()`. And `orientation_from_str()` was created so we can return the orientation number/index if we give the orientation name. So I think we have all the functionalities we need.